projects/igniteui-angular/src/lib/carousel/slide.component.ts

```diff
@@ -54,34 +57,40 @@ export class IgxSlideComponent implements OnDestroy {
      * @memberof IgxSlideComponent
      */
     @HostBinding('attr.tabindex')
-    get tabIndex() {
+    public get tabIndex() {
         return this.active ? 0 : null;
     }
 
     /**
-     * Returns the `aria-selected` of the slide.
-     *
-     * ```typescript
-     * let slide = this.slide.ariaSelected;
-     * ```
+     * @hidden
+     */
+    @HostBinding('attr.id')
+    public id: string;
+
+    /**
+     * Returns the `role` of the slide component.
+     * By default is set to `tabpanel`
      *
+     * @memberof IgxSlideComponent
      */
-    @HostBinding('attr.aria-selected')
-    public get ariaSelected(): boolean {
-        return this.active;
-    }
+    @HostBinding('attr.role')
+    public tab = 'tabpanel';
+
+    /** @hidden */
+    @HostBinding('attr.aria-labelledby')
+    public ariaLabelledBy;
 
     /**
-     * Returns the `aria-live` of the slide.
+     * Returns the `aria-selected` of the slide.
      *
      * ```typescript
-     * let slide = this.slide.ariaLive;
+     * let slide = this.slide.ariaSelected;
      * ```
      *
      */
     @HostBinding('attr.aria-selected')
-    public get ariaLive() {
-        return this.active ? 'polite' : null;
+    public get ariaSelected(): boolean {
```

Comment:
As for the aria-selected error of the slide element, it appears that [when aria-selected is used](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role), its controlled `tabpanel` should have its `aria-expanded` attribute set to true and its hidden attribute set to false, otherwise the reverse.

![image](https://user-images.githubusercontent.com/11193764/106465274-6c563280-64a2-11eb-9e50-579ca7c6b8d9.png)

Reply:
aria-selected attribute should be added only to the element with role 'tab'. 'tabpanel' element should not have such an attribute /[aria tabpanel spec](https://www.w3.org/TR/wai-aria/#tabpanel)/.. so I just removed it